tox.ini

```diff
@@ -1,11 +1,19 @@
 [flake8]
-max-line-length = 100
+max-line-length = 150
 exclude =
   lib/biokbase,
   submodules,
   */prepare_deploy_cfg.py,
-  */NarrativeRunner_server_test.py,
-  test_scripts
+  test_scripts,
+  *deploy.cfg,
+  *deploy_docker_mongo.cfg,
+  *ini,
+  *.md
+  *lib/installed_clients*,
+  */execution_engine2Impl.py*,
+  *lib/execution_engine2/db/models/transfer/*,
```

Comment:
I understand ignoring the impl file, but why the db models?

Reply:
will need to fix that

Reply:
:+1: